extensions/integration-tests/src/notebook.test.ts

```diff
@@ -6,22 +6,39 @@
 'use strict';
 
 import 'mocha';
-import assert = require('assert');
-
+import * as assert from 'assert';
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
 import { context } from './testContext';
-import { sqlNotebookContent, writeNotebookToFile, sqlKernelMetadata } from './notebook.util';
+import { sqlNotebookContent, writeNotebookToFile, sqlKernelMetadata, getFileName, pySparkNotebookContent, pySpark3KernelMetadata, pythonKernelMetadata } from './notebook.util';
 import { getBdcServer } from './testConfig';
 import { connectToServer } from './utils';
 import * as fs from 'fs';
 
 if (context.RunTest) {
-	suite('Notebook integration test suite', function() {
+	suite('Notebook integration test suite', function () {
+		setup(function () {
+			console.log(`Start "${this.currentTest.title}"`);
+		})
+		teardown(function () {
+			let testName = this.currentTest.title;
+			try {
+				let fileName = getFileName(testName);
+				if (fs.existsSync(fileName)) {
```

Comment:
@anthonydresser - in my review you mentioned not to use fs and to use pfs instead. Is that something we should be concerned about here too?

Reply:
I mentioned this in a previous review on this code. In this context it might not matter to use fs, since haulting the test thread might not matter. Although its always safer to use pfs instead.